session/clustered_index_test.go

```diff
@@ -51,6 +51,15 @@ func (s *testClusteredSuite) TestClusteredUnionScan(c *C) {
 	tk.MustExec("rollback")
 }
 
+func (s *testClusteredSuite) TestClusteredPrefixColumn(c *C) {
+	tk := s.newTK(c)
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t1(cb varchar(12), ci int, v int, primary key(cb(1)), key idx_1(cb))")
+	tk.MustExec("insert into t1 values('PvtYW2', 1, 1)")
+	tk.MustQuery("select cb from t1").Check(testkit.Rows("PvtYW2"))
+	tk.MustQuery("select * from t1").Check(testkit.Rows("PvtYW2 1 1"))
+}
+
```

Comment:
I think this test should be run in TiKV manually?

Reply:
https://github.com/pingcap/tidb/pull/14149 add mechanism make suit embedded with `testSessionSuiteBase` can run with tikv in https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tidb_ghpr_check_2/detail/tidb_ghpr_check_2/74777/pipeline/53 :D